src/main/java/org/madhuri/app/message/model/User.java

```diff
@@ -16,49 +16,48 @@
 public class User {
 
 	@Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    @Column(name = "id")
-    private Long id;
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "id")
+	private Long id;
 
-    @Column(name = "name")
-    private String name;
+	@Column(name = "name")
+	private String name;
 
-    @ManyToMany(fetch = FetchType.LAZY, mappedBy = "users")
-    private List<Channel> channels = new ArrayList<>();
+	@ManyToMany(mappedBy = "users",fetch = FetchType.EAGER)
```

Comment:
do we want to fetch all the channel for the users when we fetch users every time ?